core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -418,15 +423,21 @@ oppia.controller('CreatorDashboard', [
               });
             };
             $scope.editSuggestion = function() {
-              $scope.suggestionEditorIsShown = true;
+              $uibModalInstance.close({
+                action: ACTION_EDIT_SUGGESTION,
+                newSuggestionHtml: $scope.suggestionData.newSuggestionHtml,
+                summaryMessage: $scope.summaryMessage,
+                stateName: $scope.stateName,
+                suggestionType: $scope.suggestionType,
+                oldContent: $scope.oldContent
+              });
             };
             $scope.cancel = function() {
               $uibModalInstance.dismiss();
             };
             $scope.isEditButtonShown = function() {
               return (
-                !$scope.isNotHandled && $scope.isSuggestionRejected &&
```

Comment:
why this change?

Reply:
i think i have done some mistake , let me check and sorii for this.